src/CloudCode.js

```diff
@@ -62,19 +64,19 @@
  *
  * If you want to use beforeDelete for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
  * ```
- * Parse.Cloud.beforeDelete('MyCustomClass', (request, response) => {
```

Comment:
is it remove response in beforeDelete? 
i was used it before like this:
```
Parse.Cloud.beforeDelete('RedBookNote', async function (request, response) {
    response.success();
});

```

Reply:
In parse-server 3.0 (the the docs update is for parse-server 3.0)

Reply:
I understand this leads to a lot of confusion, I'll bring in the CLoud Code docs in parse-server instead. And just add the link here. I'll revert cloud code docs for now :)

Reply:
goood :)